commands/music/musictrivia.js

```diff
@@ -92,10 +92,25 @@ module.exports = class MusicTriviaCommand extends Command {
       const dispatcher = connection
         .play(
           ytdl(queue[0].url, {
+            filter: 'audio',
             quality: 'highestaudio',
-            highWaterMark: 1024 * 1024 * 1024
+            highWaterMark: 1 << 25
           })
         )
+        .on('error', async function(e) {
+          message.say(':x: Music Trivia has stopped could not play song!');

```

Comment:
Maybe this message should be rephrased.

Reply:
like "Trivia Stopped" ?

Reply:
Message says that "Music Trivia" has stopped and that it can't play songs, so we would need something in that style.

Reply:
Ahh.. Message is correct, just add "and".

"Music Trivia has stopped and could not play song!"